lib/created_message_queue_handler.ts

```diff
@@ -174,16 +203,16 @@ function tryGetEmailNotification(
  *          a PermanentError in case of unrecoverable error
  */
 export async function handleMessage(
-  profileModel: ProfileModel,
-  messageModel: MessageModel,
-  notificationModel: NotificationModel,
-  blobService: BlobService,
+  lProfileModel: ProfileModel,
```

Comment:
renamed local variables to not shadow the global ones

Reply:
this is a code smell, this method should be probably moved to a separate file

Reply:
I agree, there's a pivotal task with "split controllers in multiple files", probably this applies to queue handlers as well (but I wouln't do this with this PR).

Moreover, now, the error handling of the two queue handlers (processRuntimeError and the try / catch block) is very similar so probably we can do a refactor to abstract the shared code away
(ie. a function tryProcess(queueHandler) that do the error handling so we can avoid to throw in the index method).